docs/guide/best-practices.md

```diff
@@ -6,6 +6,8 @@ layout: splash
 
 - Use [boilerplate-stateful-smart-contract](https://developer.algorand.org/docs/features/asc1/stateful/#boilerplate-stateful-smart-contract) as a template for new smart-contracts.
 - Use [zero address](https://developer.algorand.org/docs/features/accounts/#special-accounts) to prevent future updates or deletion of a smart-contract.
+- Use javascript normal functions instead of arrow function while writing unit tests. Please read more
```

Comment:
Maybe we should encourage the user a bit more. `It is strongly recommended not to use arrow functions in tests due to the problem of scope of `this` ...  Use only regular function expressions. For more details go to`

Reply:
Done